package.json

```diff
@@ -39,6 +39,7 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-glob-finder": "0.0.6",
```

Comment:
I could not reproduce the `siteConfig.finder is not a function` issue so I removed it.